docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,219 @@
+An example here is that my new moderator role needs the Manage Messages permission in the #general channel.
+We will be using the :meth:`~TextChannel.set_permissions` method:
+
+.. code-block:: python3
+
+    # Let's create a PermissionOverwrite instance that has some explicit permissions set:
+    overwrites = discord.PermissionOverwrite()
+    overwrites.update(send_messages=True, send_messages_in_threads=True)
+
+    # and now we update that with `manage_messages`:
+    overwrites.manage_messages = True
+
+    # we can now set those permissions on the channel:
+    await channel.set_permissions(moderator_role, overwrites=overwrites)
+
+This short example briefly goes over PermissionOverwrite objects, let's look more into those now:
+
+Using ``PermissionOverwrite`` objects during channel creation
+--------------------------------------------------------------
+
+We can also create channels with permissions set from the start.
+Let's create a text channel where only the moderator role, and someone trying to make a report can see the channel.
+
+We need to construct a mapping of ``Role | Member: PermissionOverwrite``:
+
+.. code-block:: python3
+
+    # We can now create the overwrites mapping necessary for this, you can do it in one dictionary instantiation, or spead it across multiple lines.
+
+    # I have added type annotations here so that the reader can understand what is happeining, and so type-checkers do too:
+    overwrites: Dict[Union[discord.Role, discord.Member], discord.PermissionOverwrite] = {
+        guild.default_role: discord.PermissionOverwrite(view_channel=False, send_messages=False, read_messages=False, read_message_history=False),
+        moderator_role: discord.PermissionOverwrite(manage_messages=True),
+        member: discord.PermissionOverwrite(view_channel=True, send_messages=True, read_messages=True, read_message_history=True)
+    }
+
+    # or alternatively:
+    overwrites: Dict[Union[discord.Role, discord.Member], discord.PermissionOverwrite] = {}
+    overwrites[guild.default_role] = discord.PermissionOverwrite(view_channel=False, send_messages=False, read_messages=False, read_message_history=False)
+    overwrites[moderator_role] = discord.PermissionOverwrite(manage_messages=True)
+    overwrites[member] = discord.PermissionOverwrite(view_channel=True, send_messages=True, read_messages=True, read_message_history=True)
+
+    # We can now pass these to channel creation:
+
+    await guild.create_text_channel(name="Ticket", overwrites=overwrites)
+
+The channel created whose permissions look something like this:
+
+.. image:: /images/guide/topics/permissions/ticket_channel.png
+
+Using ``PermissionOverwrite`` objects for finer control over channels
+----------------------------------------------------------------------
+
+:class:`~PermissionOverwrite` makes granting finer access to channels easier to manipulate.
```

Comment:
It feels a touch difficult to understand what this sentence means without stopping to think about it, so maybe:
```suggestion
:class:`~PermissionOverwrite` makes controlling access to channels a much easier task.
```
would be easier to follow.